core/primitives-core/src/config.rs

```diff
@@ -5,6 +5,7 @@ use std::collections::hash_map::DefaultHasher;
 use std::hash::{Hash, Hasher};
 use strum::{Display, EnumCount};
 
+/// Describes cost of different wasm operations.
```

Comment:
I think we should make sure that we document not only the costs are a part of `VMConfig` but also limits. But also would be great if the description allowed for extending the structure without needing to change the description so that it doesn't go out of date.

So perhaps something like:

```suggestion
Configuration of the VM properties such as costs and limits.
```